datarepo-clienttests/src/main/java/scripts/testscripts/baseclasses/SimpleDataset.java

```diff
@@ -47,9 +48,25 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
 
     // create a new profile
     if (billingProfileModel == null) {
-      billingProfileModel =
-          DataRepoUtils.createProfile(
-              resourcesApi, repositoryApi, billingAccount, "profile-simple", true);
+      if (cloudPlatform.equals(CloudPlatform.GCP)) {
+        billingProfileModel =
+            DataRepoUtils.createProfile(
+                resourcesApi, repositoryApi, billingAccount, "profile-simple", true);
+      } else if (cloudPlatform.equals(CloudPlatform.AZURE)) {
+        billingProfileModel =
+            DataRepoUtils.createAzureProfile(
+                resourcesApi,
+                repositoryApi,
+                tenantId,
+                subscriptionId,
+                resourceGroupName,
+                applicationDeploymentName,
+                "azure-profile-simple",
+                billingAccount,
+                true);
+      } else {
```

Comment:
That's cool that you have the exception here but we should probably do it the other places where you are branching

Reply:
Ah okay! If we hit the exception here then the test would fail before we get to the rest of the branching logic so I had left that out, but it's probably better not to rely on that